contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -429,8 +429,9 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
          * liquidity in the pool yet. The full liquidity removal is thus skipped.
          */
         if (tokenId != 0) {
-            _removeLiquidity(1e18);
+            _removeLiquidityToFacilitateSwap(_amountToSwap, _swapWeth);
```

Comment:
🔴 `_removeLiquidity` burns the superOETHb returned from the contract. `_removeLiquidityToFacilitateSwap` does not. That could result in very fun amounts of `superOETHb` sitting on the AMO contract, increasing our TVL.

Perhaps we put a superOETHb burn  in `_ensureWETHBalance` or in `_removeLiquidity` itself?

Reply:
Note that `_addLiquidty` will call burn (and this happens near the end of the swap), but it won't burn if the weth balance is zero.

Reply:
great catch thanks, added the burn to the `_removeLiquidityToFacilitateSwap`. Will also add the functionality to tests where after the rebalance there should be no non dust WETH / OETHb left on the contract